patches/tModLoader/Terraria/Program.cs.patch

```diff
@@ -100,16 +104,41 @@
  				args = Utils.ConvertMonoArgsToDotNet(args);
  
 +			LaunchParameters = Utils.ParseArguements(args);
++			
 +			SavePath =
 +				LaunchParameters.ContainsKey("-tmlsavedirectory") ? LaunchParameters["-tmlsavedirectory"] :
-+				LaunchParameters.ContainsKey("-savedirectory") ? Path.Combine(LaunchParameters["-savedirectory"], "ModLoader") :
-+				Platform.Get<IPathService>().GetStoragePath($"Terraria{Path.DirectorySeparatorChar}ModLoader{(!BuildInfo.IsRelease ? Path.DirectorySeparatorChar + "Beta" : "")}");
++				LaunchParameters.ContainsKey("-savedirectory") ? LaunchParameters["-savedirectory"] :
++				Platform.Get<IPathService>().GetStoragePath($"Terraria");
```

Comment:
`-savedirectory` is the vanilla command, and should mirror the directory structure like you have here. `-tmlsavedirectory` is meant to specify the exact tML save path (so no preview/dev/release logic should be applied). I think this has value, otherwise we should remove `-tmlsavedirectory`